src/Attachment.tsx

```diff
@@ -8,8 +7,43 @@ import { FormatState } from './Store';
 const regExpCard = /\^application\/vnd\.microsoft\.card\./i;
 
 const YOUTUBE_DOMAIN = "youtube.com";
+const YOUTUBE_WWW_DOMAIN = "www.youtube.com";
 const YOUTUBE_SHORT_DOMAIN = "youtu.be";
+const YOUTUBE_WWW_SHORT_DOMAIN = "www.youtu.be";
 const VIMEO_DOMAIN = "vimeo.com";
+const VIMEO_WWW_DOMAIN = "www.vimeo.com";
+
+const queryStringToObject = (
+    src: string
+): any => {
+    const queryObject = {};
+    const fields = src.substr(1).split('&');
```

Comment:
The Location.search interface reutrns a '?' prefix: https://developer.mozilla.org/en-US/docs/Web/API/HTMLHyperlinkElementUtils/search